coriolis/osmorphing/base.py

```diff
@@ -156,6 +158,12 @@ def _version_supported_util(cls, version, minimum, maximum=None):
                 "Non-string version provided: %s (type %s)" % (
                     version, type(version)))
 
+        if cls.latest:
```

Comment:
I think we should move this block before `if maximum:`, otherwise any old version will pickup the latest one.